### PR TITLE
hardening/architecture_doc

### PR DESCRIPTION
**Pull request by frbattid**

### CODE REVIEW

File: flume/doc/design/architecture.md [hunk 1]
```diff
@@ -1 +1,77 @@
-TBD
+#Cygnus architecture
```
Comment (fgalan) [inline flume/doc/design/architecture.md]: Maybe the document should include the how Cygnus works in HA, from a point of view of architecture and sequence diagrams?

Comment (frbattid) [inline flume/doc/design/architecture.md]: Current HA is done by replicating a complete Cygnus agent. There is nothing, internally, specifically addressing such an issue. All the architecture variants try to addresse performance issues.

Comment (fgalan) [inline flume/doc/design/architecture.md]: In that case, I'd suggest a "High-Availability Cygnus architecture" (in parallel to the "Basic Cygnus architecture") explaining that.

Comment (frbattid) [inline flume/doc/design/architecture.md]: Fixed in https://github.com/telefonicaid/fiware-connectors/commit/d4635d3a6ec4fd7f21cec57ddf2a610061c2501c

**Comment by fgalan** [comment]
LGTM

**Pull request merged**